src/scripts/generate-unary-ops.py

```diff
@@ -0,0 +1,206 @@
+        self.test_case = ""
+
+    def configure_settings(self, options):
+        return settings.file_write_settings(
+            _GENERATED_FILENAME,
+            options.output,
+            validate.validate_numeric,
+            _constants.test_case_in_a_file)
+
+    def add_unique_test_cases(self, op): # Adds Unique test cases for some operators
+        if op in ["+", "-", "typeof"]:
+            if op in ["+", "-"]:
+                for step in ["true","false","null"]:
+                    self.expected_value = step == "true"
+                    if op == "-":
+                        self.expected_value = -self.expected_value
+                    self.expected_value = int(self.expected_value)
+                    false_values = self.generate_false_numbers(
+                        self.expected_value,
+                        _FALSE_VALUES_COUNT,
+                        _constants.min,
+                        _constants.max)
+                    false_values.append(int(step != "true"))
+                    self.test_case += self.create_declarations(step, self.TEST_VALUE_VAR_NAME.format(NUMBER = ""))
+                    self.test_case += self.create_declarations(self.expected_value, self.EXPECTED_VAR_NAME)
+                    self.test_case += self.create_declarations(false_values, self.FALSE_RESULT_VAR_NAME)
+                    self.test_case += self.create_declarations("%s%s" % (op, self.TEST_VALUE_VAR_NAME.format(NUMBER = "")), self.TEST_VALUE_VAR_NAME)
+                    self.test_case += self.create_function_call(self.TEST_VALUE_VAR_NAME.format(NUMBER = ""),
+                                                                self.EXPECTED_VAR_NAME.format(NUMBER = ""),
+                                                                self.create_false_numbers_array_string(len(false_values)),
+                                                                validate.validate_numeric_header)
+                    self.append_test_case()
+            elif op == "typeof":
+                self.test_value = 1
+                self.expected_value = "'undefined'"
+                false_values = []
+                false_values.append('string')
+                false_values.append('number')
+                false_values.append('object')
+                self.test_case += self.create_declarations(self.test_value, self.TEST_VALUE_VAR_NAME.format(NUMBER = ""))
+                self.test_case += self.create_declarations(self.expected_value, self.EXPECTED_VAR_NAME)
+                self.test_case += self.create_declarations(false_values, self.FALSE_RESULT_VAR_NAME)
+                self.test_case += "delete %s;\n" % (self.TEST_VALUE_VAR_NAME.format(NUMBER = ""))
+                self.test_case += self.create_declarations("typeof(%s)" % (self.TEST_VALUE_VAR_NAME.format(NUMBER = "")), self.TEST_VALUE_VAR_NAME)
+                self.test_case += self.create_function_call(self.TEST_VALUE_VAR_NAME.format(NUMBER = ""),
```

Comment:
This code snippet is really simulator to the previous one. Cannot we merge these cases?